js/src/main/scala/Main.scala

```diff
@@ -289,14 +306,24 @@ object Main {
           ctx += nme -> ty_sch
           declared.get(nme).foreach { sign =>
             // ctx += nme -> sign  // override with less precise declared type?
-            subsume(ty_sch, sign)(ctx, raise, TypeProvenance(d.toLoc, "def definition"))
+            subsume(ty_sch, sign)(
```

Comment:
If you're using scalafmt, please change your column width setting to at leas 100. Reformatting this needlessly is a bit ridiculous.

Reply:
Reverted.